modules/core/arrow-typeclasses/src/main/kotlin/arrow/typeclasses/MonadError.kt

```diff
@@ -160,4 +160,7 @@ interface MonadThrowFx<F> : MonadFx<F> {
     wrapReturn.startCoroutine(continuation, continuation)
     return continuation.returnedMonad()
   }
+
```

Comment:
There's a thing about this, whether we try/catch the block in Monad or not, and how this can change that behavior.

Reply:
Right. The problem I was trying to solve here was that if you overwrite `fx` in `Monad` and you overwrite `fx` in `MonadThrow` then the `fx.monad` from `Monad` gets lots.

This api seems dangerous. Should we just got for `fx`, `fxCatch`, `fxAsync` and `fxConcurrent`?

Reply:
I think every type should expose a single fx if that is an issue but multiple names is confusing

Reply:
My point was not to override it.

Reply:
We can't override because of the receiver param?